### PR TITLE
helm 5.4.0: add release notes

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
#### What this PR does

This is a draft PR with release notes. I structured them from the highlights in the [changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).

I tried to only keep the major features or potentially breaking changes. The release is light in changes, so I think that left the release notes brief. Open to feedback on including changes or excluding some.

#### Which issue(s) this PR fixes or relates to

related to https://github.com/grafana/mimir/issues/8390

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Grafana Mimir Helm chart version 5.4 release notes
+
+Grafana Labs is excited to announce version 5.4 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.13 or Grafana Enterprise Metrics (GEM) v2.13. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+A number of timeout settings have been improved upon:
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md:16]:
```suggestion
A number of timeout settings are improved:
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Fine-tuned termination grace periods for all components. This helps with ensuring that components can shut down gracefully without interrupting queries or write requests.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md:18]:
```suggestion
- Fine-tuned termination grace periods for all components. This helps to ensure that components can shut down gracefully, without interrupting queries or write requests.
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Continuous test and smoke test containers now use the Mimir image as now the continuous test code is part of Mimir.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md:19]:
```suggestion
- Continuous test and smoke test containers now use the Mimir image, as the continuous test code is now part of Mimir.
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  As a result, the `image` section in the values file now replaces `smoke_test.image` and `continuous_test.image` and the latter two have been removed.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md:20]:
```suggestion
  As a result, the `image` section in the values file now replaces `smoke_test.image` and `continuous_test.image`, and the latter two are removed.
```

File: docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Features and enhancements
+
+Notable enhancements are as follows:
+
+- Experimental support for a dedicated ruler query path. With a dedicated set of queriers, query-frontends, and
+  query-schedulers outages and slow-downs on the main query path do not affect rule evaluations.
```
Comment (tacole02) [inline docs/sources/helm-charts/mimir-distributed/release-notes/v5.4.md:27]:
```suggestion
  query-schedulers, outages and slow-downs on the main query path don't affect rule evaluations.
```

**Review by tacole02 — APPROVED** [review]
Looks great, thank you!

**Comment by dimitarvdimitrov** [comment]
Wow, that was fast, @tacole02! Thanks 🙏

**Pull request merged**